docs/docsite/rst/installation_guide/installation_distros.rst

```diff
@@ -0,0 +1,115 @@
+.. _installing_the_control_node:
+.. _from_yum:
+
+Installing Ansible on specific operating systems
+================================================
+
+Follow these instructions to install the Ansible community package on a variety of operating systems.
+
+Installing Ansible on CentOS or Fedora
```

Comment:
I'd either change the order in the title, or change the order below to match the order in the title (CentOS, Fedora, CentOS, Fedora . . . or else Fedora, CentOS, Fedora, CentOS).